docs/using-concourse/resource-types.any

```diff
@@ -168,6 +168,8 @@ before using it!
     \hyperlink{https://github.com/suhlig/concourse-rss-resource}{RSS}
   }{
     \hyperlink{https://github.com/patrickcrocker/maven-resource}{Maven Resource}
+  }{
+    \hyperlink{https://github.com/jinlee/concourse}{Counter}
```

Comment:
I think this is the wrong link, it should be https://github.com/jinless/counter-resource correct?